src/main/java/zeronote/notebooks/Notebook.java

```diff
@@ -80,13 +80,14 @@ public void addSection(Section s) {
      */
     public int findSection(String searchKey) {
         int index = 0;
+        int notFound = -1;
```

Comment:
Not sure this is necessary, but if at all, it should be a final, class-level field right?

Reply:
I just thought that returning -1 just like that would be an example of a magic literal xD that's why haha. so you mean I should initialise it right at the top, as a static final variable?